src/app/roc-data-source.ts

```diff
@@ -74,9 +74,9 @@ export abstract class RocDataSource<
         protected aetherService: AetherService,
         public bs: BasketService,
         protected target: string,
-        protected pathRoot: string,
-        protected pathListAttr: string[],
-        public indexAttr: string[],
+        protected pathRoot: string, // TODO remove use modelPath
```

Comment:
you could use 
```
/** @deprecated */ 
```
before it

Reply:
as far as I understand you can only deprecate an entire method, not single parameters